complex_envs/finite_diff_wave.py

```diff
@@ -25,34 +25,28 @@ def __init__(self,config):
         #scale the force width by system length
         self.force_width *= self.L
 
-        #driving force profile is a superposition of waves
-        self.Nwaves = config['num_waves'] 
-        self.drive_period = config['drive_period']
-        self.drive_frequency = 2.0*np.pi/self.drive_period
-
         #the lattice spacing
         self.dx = float(self.L)/float(self.Nx)
 
+        #Mesh points in space
+        self.x_mesh = np.linspace(0.0,self.L,self.Nx+1)
+        
         #the courant number
         self.C = self.c_speed *self.dt/self.dx
         self.C2 = self.C**2 #helper number
 
-        #Mesh points in space
-        self.x_mesh = np.linspace(0.0,self.L,self.Nx+1)
-
         #recalibrate the resolutions to account for rounding
         self.dx = self.x_mesh[1] - self.x_mesh[0]
         
+        #We set up the conditions of the system before warmup period
 
         #The system is always initially at rest
-        #TODO: generalize this by making V passable parameter
```

Comment:
TODOs should all be done or removed by the end of this week

Reply:
Ah, this should be removed actually since my workflow has made this obsolete.  Good catch.